src/components/InputCardBox/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {View, TextInput, StyleSheet} from 'react-native';
+
+const CardInputBox = ({text, width}) => {
+  return (
+    <View style={[styles.boxContainer]}>
+      <View style={[styles.box, {width}]}>
```

Comment:
```suggestion
      <View style={[styles.box, {width}]}>
```

这个 {width} 的 {} 有必要吗。去掉应该不会报错吧 不确定